name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/KeytradeBankPDFExtractor.java

```diff
@@ -64,17 +64,19 @@ private void addBuySellTransaction()
                 // Achat 310 HOME24 SE  INH O.N. (DE000A14KEB5) à 15,9324 EUR
                 // Vente 310 VERBIO VER.BIOENERGIE ON (DE000A0JL9W6) à 33,95 EUR
                 // Verkauf 22 SARTORIUS AG O.N. (DE0007165607) für 247 EUR
-                .section("isin", "name", "shares", "currency")
-                .match("^(Kauf|Achat|Verkauf|Vente) (?<shares>[\\.,\\d]+) (?<name>.*) \\((?<isin>[\\w]{12})\\) .* (?<currency>[\\w]{3})$")
+                .section("isin", "name", "shares", "amount", "currency")
+                .match("^(Kauf|Achat|Verkauf|Vente) (?<shares>[\\.,\\d]+) (?<name>.*) \\((?<isin>[\\w]{12})\\) .* (?<amount>[\\.,\\d]+) (?<currency>[\\w]{3})$")
                 .assign((t, v) -> {
                     t.setShares(asShares(v.get("shares")));
+                    t.setAmount(asAmount(v.get("amount")));
+                    t.setCurrencyCode(v.get("currency"));
                     t.setSecurity(getOrCreateSecurity(v));
                 })
 
                 // Ausführungsdatum und -zeit : 15/03/2021 12:31:50 CET
-                // Ordre créé à : 10/02/2021 15:23:42 CET
+                // Ordre créé à: 10/02/2021 15:23:42 CET
                 .section("date", "time")
-                .match("^(Ausf.hrungsdatum und \\-zeit|Ordre cr.. .) : (?<date>[\\d]{2}\\/[\\d]{2}\\/[\\d]{4}) (?<time>[\\d]{2}:[\\d]{2}:[\\d]{2}) .*$")
+                .match("^(Ausf.hrungsdatum und \\-zeit|Ordre cr.. .)\\s?: (?<date>[\\d]{2}\\/[\\d]{2}\\/[\\d]{4}) (?<time>[\\d]{2}:[\\d]{2}:[\\d]{2}) .*$")
```

Comment:
Hello @dimitriwauters 
I'm afraid I don't understand your changes.
In French, punctuation marks are always followed by spaces. As it is to be seen in the test files e.g. [Kauf02.txt - L13](https://github.com/buchen/portfolio/blob/53a59729777988c9e388dd08c274aac96b62db42/name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/keytradebank/Kauf02.txt#L13) and [L14](https://github.com/buchen/portfolio/blob/53a59729777988c9e388dd08c274aac96b62db42/name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/keytradebank/Kauf02.txt#L14). 
Therefore your changes are not correct, unless there is a correct PDF debug for it, which is not available yet.
In French, the grammar is that special characters are preceded by a whitespace. 
As an example: 
```
Ordre créé à : 19/10/2020 11:53:02 CET
Type d' ordre : Limit (16 EUR)
```